public/index.html

```diff
@@ -11,6 +11,12 @@
       <option disabled selected value="/">Jump to mode...</option>
       <option value="">Default</option>
       <option value="load_foods">Load Foods</option>
+      <option value="level_3_1">3/1</option>
+      <option value="level_3_3">3/3</option>
+      <option value="level_4_1">4/1</option>
+      <option value="level_4_3">4/3</option>
+      <option value="level_5_3">5/3</option>
+      <option value="level_7_6">7/6</option>
     </select>
```

Comment:
In follow up, let's make this more descriptive rather than tied to a lesson sequence that's still in flux.